### ISSUE TITLE
--cleanup removes `.DS_Store`

### ISSUE THREAD

**Issue by RhetTbull** [RhetTbull/osxphotos#1506]
`--cleanup` deletes the `.DS_Store` from the directory. This is not a problem as Finder will recreate it but OSXPhotos shouldn't do this. Exclude `.DS_Store` from cleanup. It might also be good to consider excluding any dot-file from the cleanup.

**Comment by odedia** [comment]
Makes sense to keep these, `.DS_Store` is used for preserving the display preferences in the folder (icons vs list vs details etc.), if someone is very picky on how his folders look it can get distracting.
